test/driver_test.rb

```diff
@@ -131,6 +136,77 @@
   end
 
   describe "total_revenue" do
```

Comment:
What happens if the driver has in progress trips? You must account for this in your method and test for this edge case.